workflows/artic_nanopore_nCov19.nf

```diff
@@ -22,12 +19,10 @@ workflow artic_ncov_wf {
         )
 
         // error logging
-        noreadsatall = filter_fastq_by_length.out.ifEmpty{ log.info "\033[0;33mNot enough reads in all samples, please investigate $params.output/$params.readqcdir\033[0m" }
```

Comment:
would add this to the main workflow as this can happen and users are confused if the workflow exits successfully without doing any assembly